commands/createfixpullrequests.go

```diff
@@ -309,6 +366,21 @@ func (cfp *CreateFixPullRequestsCmd) updatePackageToFixedVersion(impactedPackage
 	return packageHandler.UpdateImpactedPackage(impactedPackage, fixVersionInfo)
 }
 
+// getMinimalFixVersion that fixes the current impactedPackage
+// fixVersions array is sorted, so we take the first index, unless it's version is older than what we have now.
```

Comment:
// fixVersions array is sorted, so we take the first index, unless it's version is older than what we have now.
-->
// fixVersions is a sorted array. The function returns the first version in the array, that is larger than `impactedPackageVersion`.